packages/iov-keycontrol/src/keyring-entries/slip10.spec.ts

```diff
@@ -71,8 +87,77 @@ describe("Ed25519HdKeyringEntry", () => {
+
+      const identities = entry.getIdentities();
+      expect(identities.length).toEqual(3);
+      expect(identities[0].pubkey.algo).toEqual(Algorithm.SECP256K1);
+      expect(identities[0].pubkey.data).toEqual(newIdentity1.pubkey.data);
+      expect(identities[1].pubkey.algo).toEqual(Algorithm.SECP256K1);
+      expect(identities[1].pubkey.data).toEqual(newIdentity2.pubkey.data);
+      expect(identities[2].pubkey.algo).toEqual(Algorithm.SECP256K1);
+      expect(identities[2].pubkey.data).toEqual(newIdentity3.pubkey.data);
+    }
+  });
+
+  it("generates secp256k1 keys compatible to other tools", async () => {
+    // Test data generated by the BIP44 tool of
+    // https://iancoleman.io/bip39/#english
+
+    const entry = Slip10KeyringEntry.fromMnemonicWithCurve(Slip10Curve.Secp256k1, "mushroom faint dumb venture million true skull grab pitch mesh share tortoise");
+
+    // m/44'/0'/7'/1/0
+    const address0 = await entry.createIdentityWithPath([Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(0), Slip10RawIndex.hardened(7), Slip10RawIndex.normal(1), Slip10RawIndex.normal(0)]);
+    expect(address0.pubkey.data).toEqual(fromHex("0388557bc34cf8229fc40cffe464344e946bf5c46257e820ea1632f3acbeaa723b"));
+
+    // m/44'/0'/7'/1/1
+    const address1 = await entry.createIdentityWithPath([Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(0), Slip10RawIndex.hardened(7), Slip10RawIndex.normal(1), Slip10RawIndex.normal(1)]);
+    expect(address1.pubkey.data).toEqual(fromHex("03cf16066cbcb077cac488ad03995db1a6ad97c3f1088b59a9d5ae4ca449d7e4ad"));
+
+    // m/44'/0'/7'/1/2
+    const address2 = await entry.createIdentityWithPath([Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(0), Slip10RawIndex.hardened(7), Slip10RawIndex.normal(1), Slip10RawIndex.normal(2)]);
+    expect(address2.pubkey.data).toEqual(fromHex("02f4a71480c4f6928ad10002ab17815ea4db2a56e545e5ef74d71d7b490171db93"));
+  });
+
+  it("generates ed25519 keys compatible to Stellar", async () => {
+    // Test 1 from https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0005.md#test-cases
+    //
+    // Stellar public keys can be converted to raw ed25519 pubkeys as follows
+    // $ yarn add stellar-sdk
+    // $ node
+    // > Keypair.fromPublicKey("GDRXE2BQUC3AZNPVFSCEZ76NJ3WWL25FYFK6RGZGIEKWE4SOOHSUJUJ6").rawPublicKey().toString("hex")
+    const entry = Slip10KeyringEntry.fromMnemonicWithCurve(Slip10Curve.Ed25519, "illness spike retreat truth genius clock brain pass fit cave bargain toe");
+
+    // m/44'/148'/0'
+    const address0 = await entry.createIdentityWithPath([Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(148), Slip10RawIndex.hardened(0)]);
+    expect(address0.pubkey.data).toEqual(fromHex("e3726830a0b60cb5f52c844cffcd4eed65eba5c155e89b26411562724e71e544"));
+
+    // m/44'/148'/1'
+    const address1 = await entry.createIdentityWithPath([Slip10RawIndex.hardened(44), Slip10RawIndex.hardened(148), Slip10RawIndex.hardened(1)]);
```

Comment:
Yes, like the bip32 curve idea from the proposal sketch earlier.
I think we can just keep this name.